go.mod

```diff
@@ -3,25 +3,32 @@ module github.com/tendermint/iavl
 go 1.12
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/confio/ics23/go v0.7.0
```

Comment:
We should not depend `confio/ics23` anymore even though we use our own repo to relace it, it is not a secure lib, and cosmos are not going to use it as well.